packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -642,11 +502,94 @@ export class Relayer {
         } else throw receipt;
       } else throw receipt;
     } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Disputer", message: "Something errored disputing relay!", error });
+      this.logger.error({ at: "AcrossRelayer#Disputer", message: "Something errored disputing relay!", error });
     }
   }
 
-  private generateSlowRelayTx(deposit: Deposit, realizedLpFeePct: BN): TransactionType {
+  private async _processTransactionBatch(
+    transactions: { transaction: TransactionType | any; message: string; mrkdwn: string }[]
+  ) {
+    // Remove any undefined transaction objects or objects that contain null transactions.
+    transactions = transactions.filter((transaction) => {
+      if (!transaction) return false;
+      if (!transaction.transaction) return false;
+      return true;
+    });
+
+    if (transactions.length == 0) return;
+    if (transactions.length == 1) {
+      this.logger.debug({ at: "AcrossRelayerTxProcessor", message: "Sending transaction" });
+      await this._sendTransaction(transactions[0].transaction, transactions[0].message, transactions[0].mrkdwn);
+    }
+    if (transactions.length > 1) {
+      // The `to` field in the transaction must be the same for all transactions or the batch processing will not work.
+      // This should be a MultiCaller enabled contract.
+      const targetMultiCallerContract = new this.l1Client.l1Web3.eth.Contract(
+        getAbi("MultiCaller"),
+        transactions[0].transaction._parent._address
+      );
+
+      for (const transaction of transactions) {
+        if (targetMultiCallerContract.options.address != transaction.transaction._parent._address)
```

Comment:
Just for the sake of consistency and using as few private variables as possible.
```suggestion
        if (targetMultiCallerContract.options.address != transaction.transaction._parent.options.address)
```